docs/specs/clients/sign/rpc-methods.md

```diff
@@ -338,3 +342,87 @@ true
 | TTL     | 30       |
 | Tag     | 1115     |
 ```
+
+## Wallet Authentication RPC Methods
```

Comment:
Why do we need a dedicated section for this? `wc_sessionAuthenticated` is becoming a part of the standard Sign features suite so it should be enough to add it within all other methods

Reply:
Agreed